syntax/liquid.vim

```diff
@@ -73,6 +73,15 @@ syn region  liquidExpression matchgroup=liquidDelimiter start="{{-\=" end="-\=}}
 syn region  liquidComment    matchgroup=liquidDelimiter start="{%-\=\s*comment\s*-\=%}" end="{%-\=\s*endcomment\s*-\=%}" contains=liquidTodo,@Spell containedin=ALLBUT,@liquidExempt keepend
 syn region  liquidRaw        matchgroup=liquidDelimiter start="{%-\=\s*raw\s*-\=%}" end="{%-\=\s*endraw\s*-\=%}" contains=TOP,@liquidExempt containedin=ALLBUT,@liquidExempt keepend
 
+if b:liquid_subtype ==# 'html'
```

Comment:
This variable is not guaranteed to be defined. Use `if get(b:, 'liquid_subtype', 'html') ==# 'html'`.

Reply:
Actually that's not true, I was looking at the ftplugin which uses different logic. But I would still like this change to make it robust in the face of future refactorings.

Reply:
Ok, I've added a `exists('b:liquid_subtype')` check instead mainly because it is already on the code (but also because it's easier to understand). I can correct if it is wrong.

Reply:
That's a behavior change: My version is true when `b:liquid_subtype` is undefined and yours is false. But I think your behavior is probably the more sensible one? Leave what you have for now.